sandbox/docker-compose.yml

```diff
@@ -149,10 +149,9 @@ services:
       hornet:
         condition: service_healthy
     volumes:
-      - ./data/sandboxdb:/app/database
+      - ./data/sandboxdb/indexer:/app/database
```

Comment:
Is this really needed? As the database will be stored in `/app/database/indexer` already as defined in line 155

Reply:
As agreed in the call, sounds good to me and probably makes more sense to use the defaults of the apps and only configure the volumes 👍